server/controllers/groups.js

```diff
@@ -172,12 +151,11 @@ module.exports = function(app) {
       .then((result) => {
         const json = result.toJSON();
 
-        cb(null, {
+        return Promise.resolve({
```

Comment:
you can normally just `return { donationTotal: ... }`, will be made thenable. I actually even prefer

```
.then(result => result.toJSON())
.then(json => {
  donationTotal: ...
  ...
});
```